cmd/project_list.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/xanzy/go-gitlab"
+	lab "github.com/zaquestion/lab/internal/gitlab"
+)
+
+var projectListCmd = &cobra.Command{
```

Comment:
Would like to see some sort of test, there are many examples in the `cmd` package.